TurkishFilter/sections/specific.txt

```diff
@@ -114,6 +114,7 @@ ceofix.com##div[data-zone][style^="width:320px;height:100px;"]
 ceofix.com##div[data-zone][style^="width:300px;height:250px;"]
 ceofix.com##img.disabled[width="300"]
 ceofix.com##img.disabled[width="250"][height="90"]
+8gunhaber.com##.rightBox > div > a[target="_blank"] > img[width="330"]
```

Comment:
It's `img[width="330"]` a redundant part, just `8gunhaber.com##.rightBox > div > a[target="_blank"]`